cosmo_tester/test_suites/test_security/no_userstore_tests.py

```diff
@@ -40,7 +39,7 @@ def get_manager_blueprint_additional_props_override(self):
         shutil.copytree(src_plugin_dir,
                         self.test_manager_blueprint_path.dirname() /
                         CUSTOM_AUTH_PROVIDER_PLUGIN)
-        return {PLUGINS_PROP_PATH: self.get_plugins_settings()}
+        return {REST_PLUGIN_PATH: self.get_plugins_settings()}
```

Comment:
are you sure this works? cause now you're adding 'user_custom_auth_provider' under `node_templates.rest_service.properties` and not under `node_templates.rest_service.properties.plugins`